packages/epo-react-lib/src/atomic/Button/Button.tsx

```diff
@@ -6,7 +6,7 @@ import * as Styled from "./styles";
 export type ButtonStyleAs = "primary" | "secondary" | "tertiary" | "educator";
 
 export interface ButtonProps extends HTMLProps<HTMLButtonElement> {
-  children: ReactNode;
+  children?: ReactNode;
```

Comment:
there's a decent case for a function overload here, that if an icon is defined then other children should be optional. However it's already wrapped inside a `forwardRef` which complicates the typings